File: check_db.js
const sqlite3 = require('sqlite3').verbose();
const path = require('path');
const DB_PATH = path.join(__dirname, 'pages-functions', '.wrangler', 'state', 'v3', 'd1', 'miniflare-D1DatabaseObject', '7fcd5891cbc911dba284b564da572e81d2ba2a91a5be3afe226d9ccb3b3854a8.sqlite');
const db = new sqlite3.Database(DB_PATH);
db.get('SELECT id,user_id,generation_status,content FROM reports_tier2 WHERE generation_status="ready" ORDER BY created_at DESC LIMIT 1', (e,r) => {
  if (r) {
    const c = JSON.parse(r.content);
    console.log('ID:', r.id, 'status:', r.generation_status);
    if (c.productRecs) {
      for (const [k, v] of Object.entries(c.productRecs)) {
        console.log(k + ':', JSON.stringify(v).substring(0, 300));
      }
    }
  } else {
    console.log('No ready tier2 found');
  }
  db.close();
});

File: check_db.py
import sqlite3
db_path = r'C:\Users\yao\Documents\ChatGPT\美妆app\pages-functions\.wrangler\state\v3\d1\miniflare-D1DatabaseObject\b9a6f0e16a2e472874dc29e54456db604a5ec2fd1352c5571a33bfc8718155e4.sqlite'
conn = sqlite3.connect(db_path)
row = conn.execute("SELECT id, user_id, generation_status, source_tier1_report_id, substr(content,1,200) FROM reports_tier2 WHERE id = 't2-e2e-test'").fetchone()
print('Report:', row)
row2 = conn.execute("SELECT id, user_id, substr(report_data,1,100) FROM reports_tier1 WHERE id = 't1-e2e-test'").fetchone()
print('Tier1:', row2)

# Also check the user's ID
user = conn.execute("SELECT id, phone FROM users WHERE phone = '13900000066'").fetchone()
print('User:', user)
conn.close()
